dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpHealth.4.0.ee.10.0.mp-6.1.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Required because in `dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpHealth.4.0.ee-10.0.feature` if the feature list is set to start `mpConfig-3.0` and tolerates `3.1` issues arise when starting `microProfile-6.1`.The server will start to start `mpConfig-3.0` when `3.1` is required (even though 3.1 is tolerated).

If you explicitly set configure `mpHealth-4.0` and `mpConfig-3.1` it will work fine, but using `microProfile-6.1` will fail.

The opposite effect happens if you swap it to start `mpConfig-3.1` and tolerates `3.0`. It will run fine if `mpConfig-3.0` is set, but will fail if `microProfile-6.0` is used. It will cause it to load mpConfig-3.1 (and not accept that 3.0 is tolerated).